easy_my_coop/views/res_partner_view.xml

```diff
@@ -93,14 +97,12 @@
         </record>
         
         <record id="action_partner_cooperator_form" model="ir.actions.act_window">
-            <field name="name">Cooperator</field>
-            <field name="type">ir.actions.act_window</field>
+            <field name="name">Cooperators</field>
             <field name="res_model">res.partner</field>
             <field name="view_type">form</field>
             <field name="view_mode">kanban,tree,form</field>
-            <field name="domain">[('cooperator','=',True),('member','=',True),('old_member','=',False)]</field>
+            <field name="domain">['|', ('cooperator','=',True), '|', ('member','=',True),('old_member','=', True)]</field>

```

Comment:
I don't agree it should only display the effectif member. Not the old ones. Cooperator is a legacy field that just flag the res partner taken in charge by the easy my coop module

Reply:
Yes I agree but there is the default filter `'search_default_cooperator':1` on the next line. It points to:

```python
<filter string="Cooperators" name="cooperators" domain="[('member','=',1)]" help="Cooperator Partners"/>
```

That way, the default view shows the effective cooperators but allows to navigate through all partners linked to the app if the user removes the filter.